src/main/java/org/folio/search/service/consortium/ConsortiumSearchHelper.java

```diff
@@ -40,6 +40,15 @@ public class ConsortiumSearchHelper {
   private final FolioExecutionContext folioExecutionContext;
   private final ConsortiumTenantService consortiumTenantService;
 
+  public QueryBuilder filterQueryForNotSharedActiveAffiliation(QueryBuilder query, String tenantId, String resource) {
+    var centralTenantId = consortiumTenantService.getCentralTenant(tenantId).orElse("consortium");
+    var boolQuery = prepareBoolQueryForActiveAffiliation(query);
+
+    addActiveAffiliationClauses(boolQuery, tenantId, centralTenantId, resource, false);
+
+    return boolQuery;
+  }
+
```

Comment:
`consortium` - is not always name of central tenant

Reply:
fixed